### PR TITLE
Varrene/entry task

### PR DESCRIPTION
**Pull request by aqyra**
### Summary 
Added profile page for entry task

### Detail
Added ProfileContent in pageContent, profile in pages, and edited Navbar links in components

### Pictures / Gif 
-

### Checks 
-

### CODE REVIEW

File: src/pages/profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NextPage } from "next";
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const Profile: NextPage = () => {
+  return (
+    <Page
+      bgImage="varrene"
```
Comment (bernarduskrishna) [inline src/pages/profile/index.tsx:9]: Hem what does this line do? Right now putting `bgImage="varrene"` won't give you any background image. What should you do in order to add a background image?

File: src/pageContent/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  
+    return (
+<>
+    <div className={`flex flex-col items-center overflow-hidden`}>
+        <div className={S.StoryImage}>
+            <Image src="/assets/images/Varrene.jpg" alt="Varrene" 
+                width={350} height={350} />
+        </div>
+        
+        <div className={S.EmailUsSection}>
```
Comment (bernarduskrishna) [inline src/pageContent/Profile/index.tsx:54]: Does the className matter? Explain why.

**Review by bernarduskrishna — COMMENTED** [review]
Good job. For the pull request you can add more details so that we can see the summary of what you have done. Right now you are using the default one haha.

File: src/pageContent/Profile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        `${content}\n\n` +
+        `Thank you.\n\n` +
+        `Best regards,\n\n` +
+        `${name}\n`;
+  
+      location.href =
+        `mailto:varrene@u.nus.edu` +
+        `&subject=${encodeURIComponent(subject)}` +
+        `&body=${encodeURIComponent(body)}`;
+    };
```
[the remark this answers is not in the document]
Comment (aqyra) [reply src/pageContent/Profile/index.tsx:44]: &subject to ?subject ?

File: src/pageContent/Profile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  
+    return (
+<>
+    <div className={`flex flex-col items-center overflow-hidden`}>
+        <div className={S.StoryImage}>
+            <Image src="/assets/images/Varrene.jpg" alt="Varrene"
```
[the remark this answers is not in the document]
Comment (aqyra) [reply src/pageContent/Profile/index.tsx:50]: to give a text description of the image

File: src/pageContent/Profile/styles.tsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const EmailUsSection = `min-h-screen max-h-full w-screen flex flex-row justify-center items-center py-8 px-3`;
+
+export const HeaderText = `font-bold text-4xl lg:text-3xl text-gray-900 text-center`;
```
Comment (bernarduskrishna) [inline src/pageContent/Profile/styles.tsx:3]: If i delete the `text-center` what will happen? Explain why.

File: src/components/Navbar/links.tsx [hunk 6]
```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  { title: "Profile", slug: "profile"}
```
[the remark this answers is not in the document]
Comment (aqyra) [reply src/components/Navbar/links.tsx:13]: title is the page name and slug shortens the page directory / url

**Pull request closed without merging**